pkg/controller/directimagemigration/migrate.go

```diff
@@ -42,8 +44,11 @@ func (r *ReconcileDirectImageMigration) migrate(imageMigration *migapi.DirectIma
 	err := task.Run()
 	if err != nil {
```

Comment:
Do we want to handle more errors here in the future?

Reply:
I think a lot of errors that are non-fatal are caught early within phase logic and not passed back to this point. If an error makes it here, it's considered critical and fails the migration. 

Did you have other types of errors in mind? Want to make sure I've got your point.

Reply:
I am pretty sure that is not the case in actuality, (i.e NotFound errors or Service Errors from Kube)